decentraland/screens/helpers.js

```diff
@@ -50,20 +52,21 @@ export const getContracts = () => {
   return contracts;
 };
 
-export const approveManaSpending = async fromAccount => {
+export const addressesAreEqual = (address1, address2) => {
+  return address1.toUpperCase() === address2.toUpperCase();
+};
+
+export const approveManaSpending = async (fromAccount, onSuccess) => {
   const contracts = getContracts();
   const { manaContract, marketplaceContract } = contracts;
   const toAddress = marketplaceContract.getAddress();
   manaContract.setWallet(fromAccount);
   const action = manaContract.approve(toAddress, TEN);
   await action.waitForNonceToUpdate();
+  onSuccess();
```

Comment:
If we're going to use the `onSuccess` callback, we could probably get rid of the `await` of `waitForNonceToUpdate` and use our listening style?

Reply:
Sure!

Reply:
Hmmm.. on my last change we aren't using callback no more:
```javascript
      const fund = fundAccount(accountAddress);
      const approve = approveManaSpending(account);
      await Promise.all([fund, approve]);
      showInfo(`Account created and funded!`);
```

Reply:
Ah ok. Well we can continue the discussion thread in the other thread and likely move it off GitHub since it's a larger topic. That approach looks good for now.

Reply:
Or yeah, like we said in the other thread, we can move the discussion to this issue https://github.com/tasitlabs/tasit/issues/151

Resolving this